.github/workflows/backend.yml

```diff
@@ -107,6 +107,57 @@ jobs:
       - name: Handle artifacts
         uses: ./.github/actions/artifacts
 
+  backend-test-in-silo-modes:
+    needs: files-changed
+    if: needs.files-changed.outputs.backend == 'true'
+      && (
+            contains(github.head_ref, '/hc-')
+         || contains(github.head_ref, 'hybrid-cloud')
+         )
+    continue-on-error: true
+    timeout-minutes: 60
+    name: Runs backend tests marked for control silo mode, as part of hybrid cloud.
```

Comment:
this is going to almost always be cut off in display -- I might do something like "backend tests, hybrid cloud" or something a bit shorter

Reply:
👍